dts/arm/nordic/nrf54h20_enga_cpurad.dtsi

```diff
@@ -47,5 +47,9 @@ cpuppr_vevif: &cpuppr_vevif_remote {};
 };
 
 &grtc {
-	interrupts = <109 NRF_DEFAULT_IRQ_PRIORITY>;
+	owned-channels = <7 8 9 10 11 12 13 14>;
+	child-owned-channels = <8 9 10 11 12>;
+	nonsecure-channels = <8 9 10 11 12>;
+	interrupts = <109 NRF_DEFAULT_IRQ_PRIORITY>,
+		     <108 NRF_DEFAULT_IRQ_PRIORITY>;
 };
```

Comment:
In what way is this configuration SoC-specific?
Should those channels always be reserved in every conceivable nRF54H20 project?

If this is just default configuration (not mandatory) then I think it should be moved to board files. SoC files usually just describe the hardware.

Reply:
As of now that is common for all board that have nRF54h20. If desired solution is ready then this will be applied when a library that requests it is enabled.